Modules/Microsoft365DSC/DSCResources/MSFT_EXODataClassification/MSFT_EXODataClassification.schema.mof

```diff
@@ -0,0 +1,17 @@
+[ClassVersion("1.0.0.0"), FriendlyName("EXODataClassification")]
+class MSFT_EXODataClassification : OMI_BaseResource
+{
+    [Key, Description("The Identity parameter specifies the data classification rule that you want to modify.")] String Identity;
+    [Write, Description("The Description parameter specifies a description for the data classification rule. You use the Description parameter with the Locale and Name parameters to specify descriptions for the data classification rule in different languages. ")] String Description;
+    [Write, Description("The Fingerprints parameter specifies the byte-encoded document files that are used as fingerprints by the data classification rule.")] String Fingerprints[];
+    [Write, Description("IsDefault is used with the Locale parameter to specify the default language for the data classification rule.")] Boolean IsDefault;
+    [Write, Description("The Locale parameter adds or removes languages that are associated with the data classification rule.")] String Locale;
+    [Write, Description("The Name parameter specifies a name for the data classification rule. The value must be less than 256 characters.")] String Name;
+    [Write, Description("Specifies if this Outbound connector should exist."), ValueMap{"Present","Absent"}, Values{"Present","Absent"}] String Ensure;
```

Comment:
Please update description since this refers to a DataClassification rule and not an outbound connector

Reply:
Oops missed that one :) 
It's all ready now